docs/decisions/app/0005-server-rendering.md

```diff
@@ -0,0 +1,73 @@
+## Context and Problem Statement
+
+Next.js provides multiple ways of rendering a website, which have different commands: `next start` vs `next export`. For the platform, we need to determine which should be the default.
+
+## Considered options
+
+1.  Server rendering (`next start`): Generates the full HTML markup for a page on the server in response to navigation. Minimal client-side JS is possible, even for personalized sites.
```

Comment:
i'm confused by the wording, do you mean "it's possible to make a website that only uses minimal JS, even for personalized sites", or do you mean that "client-side JS possibilities are minimal"? It kind of sounds like the latter but I assume you mean the former

Reply:
Yeah, I mean the former. I think I'll just remove that sentence.